ecs-cli/modules/cli/local/converter/converter.go

```diff
@@ -281,7 +281,7 @@ func convertLogging(logConfig *ecs.LogConfiguration) *composeV3.LoggingConfig {
 	driver := aws.StringValue(logConfig.LogDriver)
 	for _, unsupported := range unsupportedDrivers {
 		if driver == unsupported {
-			log.Warningf("%s will not be supported", unsupported)
+			log.Infof("%s log driver is ignored when running locally. Tasks will default to json-file instead. This can be changed in your compose override file.", unsupported)
```

Comment:
nit: "json-file" is a const that's defined in [jsonFileLogDriver](https://github.com/aws/amazon-ecs-cli/blob/ecs-local/ecs-cli/modules/cli/local/converter/override.go#L25) can we use that?